cloudferrylib/os/image/glance_image.py

```diff
@@ -135,7 +135,7 @@ def get_ref_image(self, image_id):
         try:
             return self.glance_client.images.data(image_id)._resp
         except exc.HTTPInternalServerError as e:
-            LOG.error("There is an error occurred while trying to reach "
+            LOG.debug("There is an error occurred while trying to reach "
```

Comment:
Yeah, it is. But the main point here was to not see this error message in the output and logs (cause it has big scary red traceback, that can confuse someone) during the migration process. But instead of it, we will be able to see this information if we enable Debug mode.